resource/base/segments_traits_operators.md

```diff
@@ -65,6 +65,11 @@ Traits use the following syntax: `<segment>:<trait>`
 
 | **Trait** | **Description** | **Valid Types** |
 | `base:universal_access` | Set a resource's **Universal Access** level. | `str` |
+| `base:audit_alter` | Set a resource's audit rules to audit **Alter** accesses of the specified type. | `str` |
+| `base:audit_control` | Set a resource's audit rules to audit **Control** accesses of the specified type. | `str` |
+| `base:audit_none` | Clear a resource's **Audit Rules**. | `True` |
```

Comment:
```markdown
| `base:audit_alter` | Set a resource's **Auditing Rules** to audit **Alter** accesses of the specified type of **Access Attempt**. | `str` |
| `base:audit_control` | Set a resource's **Auditing Rules** to audit **Control** accesses of the specified type of **Access Attempt**. | `str` |
| `base:audit_none` | Remove all of a resource's **Auditing Rules**. | `True` |
```

Reply:
Make the same changes to lines 71 and 72.

Reply:
changed.